vector/src/main/java/im/vector/activity/VectorMediaViewerActivity.java

```diff
@@ -338,9 +338,6 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             if (requestCode == PermissionsToolsKt.PERMISSION_REQUEST_CODE) {
                 // Request comes from here
                 onAction(mPendingPosition, mPendingAction);
-            } else if (requestCode == PermissionsToolsKt.PERMISSION_REQUEST_OTHER) {
```

Comment:
`PERMISSION_REQUEST_OTHER` could be removed too.

Reply:
Do you mean `PERMISSION_REQUEST_CODE` in the line above? I think that might be used for the normal download action.

Reply:
No, I mean removing `PERMISSION_REQUEST_OTHER` from file `PermissionsTools.kt`, sorry, I was not clear